src/components/SearchTracePage/TraceSearchForm.js

```diff
@@ -154,14 +167,24 @@ export function TraceSearchFormImpl(props) {
         )}
 
         <div className="search-form--tags field">
-          <label htmlFor="tags">Tags</label>
-          <div className="ui input">
-            <Field
-              name="tags"
-              type="text"
-              component="input"
-              placeholder="http.status_code:400|http.status_code:200"
+          <label htmlFor="tags">
+            Tags{' '}
+            <Popup
+              on="click"
+              trigger={<i className="info circle icon grey" />}
+              content={
+                <span>
+                  Values should be in the{' '}
+                  <a href="https://brandur.org/logfmt" rel="noopener noreferrer" target="_blank">
+                    logfmt
+                  </a>{' '}
```

Comment:
perhaps we could give an example, save people a click

> e.g. `db.statement="select * from User"`. Use space for conjunctions: `error=true http.status_code:503`

Reply:
The placeholder has two examples. But, I can add a few to the info tooltip, including one that uses quotes.

Reply:
Nevermind, I see your note now. I'll add that to the tooltip :+1: